app/screens/interactive_dialog/interactive_dialog.test.js

```diff
@@ -106,4 +106,38 @@ describe('InteractiveDialog', () => {
         expect(baseProps.actions.submitInteractiveDialog).toHaveBeenCalledTimes(1);
         expect(baseProps.actions.submitInteractiveDialog).toHaveBeenCalledWith(dialog);
     });
+
+    describe('generic error message', () => {
+        test('should show error when submit returns an error', async () => {
+            const props = {
+                ...baseProps,
+                actions: {
+                    ...baseProps.actions,
+                    submitInteractiveDialog: async () => ({
+                        data: {error: 'This is an error message.'},
+                    }),
+                },
+            };
+
+            const wrapper = shallow(
+                <InteractiveDialog {...props}/>
+            );
+            wrapper.instance().scrollView = {current: {scrollTo: jest.fn()}};
```

Comment:
The `ScrollView` isn't available to interact with in the test environment, so I mocked it instead.  Is there a better way to do this?

Reply:
If you use `mount` instead of `shallow` refs should be available, but we are not really trying to determine if the scrollview is scrolling here, so I guess this is fine. Maybe @prapti can provide a different approach?